torch/csrc/distributed/rpc/init.cpp

```diff
@@ -49,6 +49,9 @@ PyObject* rpc_init(PyObject* /* unused */) {
           .def(py::init<std::string,
                         std::unordered_map<std::string, int>,
                         std::shared_ptr<::c10d::ProcessGroup>>())
+          .def("get_id",
+               &ProcessGroupAgent::getId,
+               py::call_guard<py::gil_scoped_release>())
```

Comment:
This reads as if `get_id` returns the ID of the agent itself.

The ID should be globally consistent and be serializable, so that it can be passed to a remote function as indirection (e.g. A tells B to call C). Then, we will need another function that returns the ID of the agent itself, which would likely be called `get_id`. What do you think of renaming this to `get_worker_id`, to indicate it takes an argument and does not necessarily apply to the local instance?

Reply:
good point! Will make the change.